fuzzing/private/binary.bzl

```diff
@@ -121,6 +123,9 @@ def _fuzzing_binary_impl(ctx):
             corpus_dir = ctx.file.corpus,
             dictionary_file = ctx.file.dictionary,
             engine_info = ctx.attr.engine[FuzzingEngineInfo],
+            # If used within rules_fuzzing, the workspace_name of the binary can
+            # be the empty string.
```

Comment:
It would be, as we need to create a distinct `binary_workdir`. But this situation should not happen: If used from another repository with not workspace name, `binary.label.workspace_name` will be `__main__` based on my experiments. The only case where it is the empty string is when used from `rules_fuzzing` itself, which we work around by substituting `ctx.workspace_name` in that case.